chrome/content/zotero/elements/attachmentsBox.js

```diff
@@ -95,6 +95,13 @@
 			return this._preview;
 		}
 
+		get _renderDependencies() {
+			if (this.item?.isRegularItem()) {
+				return [...super._renderDependencies, this.item.getAttachments().join(',')];
```

Comment:
Is this really what you mean? If so, can you add a comment explaining it? Because the string concatenation looks really weird here.

Reply:
Yeah, actually it doesn’t matter if it’s a string or something else. It just needs to be something comparable to tell the attachment of the regular item is added/removed. I concat it as a string instead appending all the ids because we might want to keep the array length fixed.

Reply:
Comment added